### ISSUE TITLE
Theft: change from clean money to black money

### ISSUE THREAD

**Issue by m0iises19** [KRILLE123/esx-kr-advanced-shops#19]
Would there be any way for robberies to give you the black money?

**Comment by DiEgUiTrUx97** [comment]
yes, just search in server side code a code like "xPlayer.giveMoney()" and change it to "xPlayer.giveAccountMoney("black_money")
